zmsadmin/tests/Zmsadmin/WorkstationProcessCalledTest.php

```
@@ -95,4 +104,52 @@ public function testWithStatusProcessing()
+                ]
+            ]
+        );
+        $response = $this->render(['id' => 100044], [], []);
+        $this->assertStringContainsString('client-confirm-call-other', (string)$response->getBody());
+        $this->assertStringContainsString('nextprocess=100044', (string)$response->getBody());
+        $this->assertEquals(200, $response->getStatusCode());
+    }
+
+    public function testConflictRejectsOutOfScopeProcessWhenClusterWideCallDisabled()
+    {
+        \App::$allowClusterWideCall = false;
```

PHPStan (2.2.6)

[error] Access to an undefined static property App::$allowClusterWideCall.


(staticProperty.notFound)